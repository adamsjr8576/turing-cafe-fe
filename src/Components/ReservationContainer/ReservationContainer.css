.main-container {
  display: flex;
  align-items: center;
  justify-content: space-around;
  flex-wrap: wrap;
  width: 100%;
  overflow: auto;
  margin-top: 25px;
}

.select-container {
  width: 100%;
}

.order-select {
  width: 20%;
  height: 30px;
  font-size: 1em;
  border-radius: 10px;
  outline: 0;
}
