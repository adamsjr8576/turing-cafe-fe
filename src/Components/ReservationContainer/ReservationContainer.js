import React from 'react';
import './ReservationContainer.css';
import ReservationCard from '../ReservationCard/ReservationCard.js'

const ReservationContainer = ({ reservations }) => {
  const reservationCards = reservations.map(reservation => {
    return (
      <ReservationCard
        date={reservation.date}
        id={reservation.id}
        name={reservation.name}
        number={reservation.number}
        time={reservation.time}
        key={reservation.id}
      />
    )
  })
  return(
    <main className='main-container'>
      {reservationCards}
    </main>
  );
}

export default ReservationContainer;
