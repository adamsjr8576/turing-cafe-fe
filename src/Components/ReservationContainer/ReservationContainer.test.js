import React from 'react';
import ReactDOM from 'react-dom';
import ReservationContainer from './ReservationContainer';
import { shallow } from 'enzyme';

describe('ReservationContainer', () => {

  it('should match the snapshot with all data passing correctly', () => {
    const wrapper = shallow(<ReservationContainer
      reservations={[{date:'12/5', name:'John Adams', number:4, time:'4:00'}]}
      />);

    expect(wrapper).toMatchSnapshot();
  })

});
