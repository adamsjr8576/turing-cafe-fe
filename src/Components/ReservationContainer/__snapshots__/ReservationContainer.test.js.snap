// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReservationContainer should match the snapshot with all data passing correctly 1`] = `
<main
  className="main-container"
>
  <ReservationCard
    date="12/5"
    name="John Adams"
    number={4}
    time="4:00"
  />
</main>
`;
